x-pack/plugins/observability_solution/infra/public/components/shared/alerts/links/create_alert_rule_button.tsx

```diff
@@ -10,20 +10,21 @@ import { EuiButtonEmpty } from '@elastic/eui';
 
 export interface LinkToAlertsRuleProps {
   onClick?: () => void;
+  dataTestSubj?: string;
 }
 
-export const LinkToAlertsRule = ({ onClick }: LinkToAlertsRuleProps) => {
+export const CreateAlertRuleButton = ({ onClick, dataTestSubj }: LinkToAlertsRuleProps) => {
```

Comment:
nit: Would it be better to use the kebab-case convention for the `data-test-subj` prop to keep consistency with other components?

```suggestion
  ['data-test-subj']?: string;
}

export const CreateAlertRuleButton = ({
  onClick,
  ['data-test-subj']: dataTestSubj,
}
```

Reply:
sure, let me update it